src/main/java/org/everit/email/store/ri/ecm/internal/EmailStoreComponent.java

```diff
@@ -0,0 +1,123 @@
+/*
+ * Copyright (C) 2011 Everit Kft. (http://www.everit.org)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.everit.email.store.ri.ecm.internal;
+
+import java.util.Hashtable;
+
+import org.everit.blobstore.Blobstore;
+import org.everit.email.store.EmailStore;
+import org.everit.email.store.ri.EmailStoreImpl;
+import org.everit.email.store.ri.ecm.EmailStoreConstants;
+import org.everit.osgi.ecm.annotation.Activate;
+import org.everit.osgi.ecm.annotation.Component;
+import org.everit.osgi.ecm.annotation.ConfigurationPolicy;
+import org.everit.osgi.ecm.annotation.Deactivate;
+import org.everit.osgi.ecm.annotation.ManualService;
+import org.everit.osgi.ecm.annotation.ServiceRef;
+import org.everit.osgi.ecm.annotation.attribute.StringAttribute;
+import org.everit.osgi.ecm.annotation.attribute.StringAttributes;
+import org.everit.osgi.ecm.component.ComponentContext;
+import org.everit.osgi.ecm.extender.ECMExtenderConstants;
+import org.everit.persistence.querydsl.support.QuerydslSupport;
+import org.everit.transaction.propagator.TransactionPropagator;
+import org.osgi.framework.Constants;
+import org.osgi.framework.ServiceRegistration;
+
+import aQute.bnd.annotation.headers.ProvideCapability;
+
+/**
+ * ECM component for {@link EmailStore} interface based on Email Store reference implementation.
+ */
+@Component(componentId = EmailStoreConstants.SERVICE_PID,
+    configurationPolicy = ConfigurationPolicy.OPTIONAL,
+    label = "Everit Email Store Component",
+    description = "ECM component for Email Store")
+@ProvideCapability(ns = ECMExtenderConstants.CAPABILITY_NS_COMPONENT,
+    value = ECMExtenderConstants.CAPABILITY_ATTR_CLASS + "=${@class}")
+@StringAttributes({
+    @StringAttribute(attributeId = Constants.SERVICE_DESCRIPTION,
+        defaultValue = EmailStoreConstants.DEFAULT_SERVICE_DESCRIPTION,
+        priority = EmailStoreComponent.P00_SERVICE_DESCRIPTION,
```

Comment:
Do not have the priority number in the constant name of the component. Reprioritizing the ocmponent attributes should not break backward compatibility.